ansible/deploy_netvault.yml

```diff
@@ -0,0 +1,24 @@
+- name: "Install Netvault Client"
+  hosts: pg_standby
+  become: true
+  tasks:
+    - name: "Download Netvault"
+      get_url:
+        url: "{{ netvault_url }}"
```

Comment:
where is `netvault_url`  defined?

Reply:
should be defined in vault, i think but i am not sure this is a long-lasting variable, so maybe extra_vars?

Reply:
Is it a 'private' url? If not can just put it in here directly since this is only going to be run on ICDS.